docs/src/guides/quarkus/elasticsearch.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "How to Deploy Quarkus on Platform.sh with Elasticsearch"
+sidebarTitle: "Elasticsearch"
+weight: -110
+layout: single
+toc: false
+description: |
+    Configure a Quarkus application with Elasticsearch.
+---
+
+Quarkus provides two ways of accessing Elasticsearch: via the lower level `RestClient` or via the `RestHighLevelClient` we will call them the low level and the high level clients.
+
+To move your Quarkus application with Elasticsearch to Platform.sh, it needs to change two configurations files, if you compare it with [a Quarkus project from scratch](_index.md).
```

Comment:
@Crell @chadwcarlson 

I did not like this point here.
I want to reduce the number of duplicated information; that why I have a getting start page where an application will run even without the services. Then, those specialized pages will append the services and further configurations to work with Platform.sh
If you have any suggestions, recommendations, please, let me know.